plugins/modules/filesystem.py

```diff
@@ -197,6 +198,125 @@ def fs_state(module, filesystem):
     return False
 
 
+def check_attr_change(module, filesystem):
+    """
+    Determines if changes will be made on the filesystem.
+    param module: Ansible module argument spec.
+    param filesystem: filesystem name.
+    return: True - changes will be made on the filesystem / False = filesystem will remain unchanged
+    """
+
+    cmd = "lsfs -cq %s" % filesystem
+    rc, stdout, stderr = module.run_command(cmd)
+    if rc != 0:
+        msg = "Failed to fetch current attributes of '%s'. cmd - '%s'" % (filesystem, cmd)
+        module.fail_json(msg=msg, rc=rc, stdout=stdout, stderr=stderr)
+
+    all_attr = stdout.splitlines()
+    old_attr = all_attr[1].split(":")
```

Comment:
probably show stdout or old_attr format here for better understanding

Reply:
like a comment in the code that will go be published in the official repo? or just a comment for the review?

Reply:
I think comment in the code should be file, just high level summery, it is my 2 cents

Reply:
okay, how about this comment"
`
# old_attr is a list containing the current values of attributes such as:
# mount group, size, permissions, auto mount, and accounting subsystem.
# old_ext_attr is a dictionary containing the current values of the extended
# attributes of the filesystem depending on the filesystem type.
`

Reply:
old_attr[0] = mount group
old_attr[1] = size
...
or 
old_attr contains mount group, size, permission, auto mount (it should be in order)